src/test/e2e/tests/specs.test.ts

```diff
@@ -19,6 +20,9 @@ let searchPage: SearchPage;
 let searchResultsPage: SearchResultsPage;
 let subscriptionManagementPage: SubscriptionManagementPage;
 let liveCaseCourtSearchControllerPage: LiveCaseCourtSearchControllerPage;
+let liveCaseStatusPage: LiveCaseStatusPage;
+
+const validCourtName = 'Ailibugai Court';
```

Comment:
small thing, wont request changes for it but, can you move this into the describe for the flow its used in?